SCsub

```diff
@@ -13,8 +13,12 @@ protobuf_include_dir = [
 ]
 
 protobuf_lib_dir = Dir("libprotobuf/lib").path
-env.Append(LINKFLAGS="-L" + protobuf_lib_dir)
-env.Append(LIBS="protobuf")
+if env["platform"] == "windows":
+    env.Append(LINKFLAGS="-LIBPATH:" + protobuf_lib_dir)
```

Comment:
ohhh, so it's a `-` not `/`? The microsoft docs says it's a slash XD

Reply:
It replace it by /, do you want I put / instead ?

Reply:
Yeah, I think it's better to use `/`.

Reply:
done.